code/__HELPERS/unsorted.dm

```diff
@@ -2059,3 +2059,38 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			return slot_wear_pda
 		if(SLOT_TIE)
 			return slot_tie
+
+
+/**
+  * HTTP Get (Powered by RUSTG)
+  *
+  * This proc should be used as a replacement for [world.Export()] due to an underlying issue with it.
+  * See: https://www.byond.com/forum/post/2772166
+  * The one thing you will need to be aware of is that this no longer wraps the response inside a "file", so anything that relies on a file2text() unwrap will need tweaking.
+  * RUST HTTP also has better support for HTTPS as well as weird quirks with modern webservers.
+  * Returns an assoc list that follows the standard [world.Export()] format (https://secure.byond.com/docs/ref/index.html#/world/proc/Export), with the above exception
+  *
+  * Arguments:
+  * * url - URL to GET
+  */
+/proc/HTTPGet(url)
```

Comment:
TGS DMAPI still uses `world.Export()`. 

However, I  cannot replace that due to the fact its part of the DMAPI which is kept in sync with the TGS master repo. On the bright side, the issue is with `world.Export()` data getting sent to the wrong `world.Export()` calls.

If theres only one call, it should be fine. .....right?